plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/StandardProjectWizard.java

```diff
@@ -31,6 +35,13 @@ public void addPages() {
 
   @Override
   public boolean performFinish() {
+    if (config.getCloudSdkLocation() == null) {
+      File location = CloudSdkPrompter.getCloudSdkLocation(new SameShellProvider(getShell()));
+      if (location == null) {
+        return false;
```

Comment:
If I read this right, this may leave config.getCloudSdkLocation as null, in which case we need to be ready for this elsewhere.